docs/pricing/index.mdx

```diff
@@ -76,7 +76,36 @@ Tracing is enabled by and will be billed for in spans.
 
 #### Cron Monitors Pricing
 
-Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a pay-as-you-go budget that can be shared across all event types.
+All Sentry plans include **one cron monitor**. To activate additional monitors, set up a **Pay-As-You-Go (PAYG) budget**. Additional monitors can't be purchased in advance; they are only available through your PAYG budget, which can be shared across all event types.
+
+**Key Points:**
+
+- **Deactivating or Deleting Monitors**:
+  - Deactivated or deleted monitors will **count** towards your billing quota if they were previously active in the current billing period. Otherwise, they don't count towards your billing quota.
+  - Their quota becomes **reusable** within the **same billing period**.
+- **Activating Monitors**:
+  - Sentry first uses any **reusable quota** from monitors deactivated or deleted in the current billing period.
+  - If none is available, your **reserved quota** or **PAYG budget** is used.
+
+<Alert>
+
+**Important:**
+
+- **Quota Reuse Limitations**:
+  - Only available **within the same billing period**.
+  - Applies to monitors that were previously active and billed.
+- **Billing Period Reset**:
+  - At the start of a new billing period, all quotas reset.
```

Comment:
Will the bullet point "At the start of a new billing period, all quotas reset." be confusing with the following section elaborating on how new billing periods are handled? The quota does re-set but if there is sufficient budget available, the assigned seats are recreated as a part of invoicing so it can _seem_ like the quota isn't resetting even though it is. 

Maybe just saying that the reusable quota doesn't carry over to new billing periods is clearer ?

Reply:
@mbauer404 Good point. I added that in.